lib/domain/usecase/load_more_internal_contacts.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:dartz/dartz.dart';
+import 'package:fluffychat/app_state/failure.dart';
+import 'package:fluffychat/di/global/get_it_initializer.dart';
+import 'package:fluffychat/domain/app_state/contact/get_contacts_success.dart';
+import 'package:fluffychat/domain/app_state/contact/get_network_contact_failed.dart';
+import 'package:fluffychat/domain/app_state/contact/get_network_contact_success.dart';
+import 'package:fluffychat/domain/model/contact/contact_query.dart';
+import 'package:fluffychat/domain/repository/contact_repository.dart';
+
+class LoadMoreInternalContacts {
+  final _contactRepository = getIt.get<ContactRepository>();
+
+  Stream<Either<Failure, GetContactsSuccess>> execute({int limit = 30, int? offset = 0}) async* {
+    try {
+      final contacts = await _contactRepository
+        .loadMoreContact(
+          query: ContactQuery(keyword: ''),
```

Comment:
- Can be optional? WDYT?

Reply:
take it in later, we can reuse `loadmore` for search also here. But ignore it now